README.md

```diff
@@ -45,4 +45,8 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./LICENSE) licensed.
\ No newline at end of file
+This project is [MIT](./LICENSE) licensed.
+
+## Live version
+
+https://bill7pearl.github.io/portfolio
```

Comment:
[Optional] I suggest that you take the link to the upper section of the readme file where it'll be easily visible. The section `Live Demo` in the Microverse README template is more appropriate.

Reply:
@emmiiorji Alright sure thing, thank you.